src/_includes/steering-committee.html

```diff
@@ -0,0 +1,73 @@
+<div class="container">
+  <div class="row">
+    <div class="col-sm-4">
+      <h5>Hack Night Lead:</h5>
+      <div>
+        <img
+          class="steeering-committee-image"
+          alt="Photo of Colin King-Bailey"
+          src="/assets/images/steering-committee/Colin.jpg"
+        />
+
+        <p class="steeering-committee-name">
+          Colin King-Bailey
+        </p>
+      </div>
+    </div>
+    <div class="col-sm-4">
+      <h5>Treasurer/Recorder:</h5>
+      <div>
+        <img
+          class="steeering-committee-image"
+          alt="Photo of Mike Ubell"
+          src="/assets/images/steering-committee/Mike.jpg"
+        />
+
+        <p class="steeering-committee-name">
+          Mike Ubell
+        </p>
+      </div>
+    </div>
+    <div class="col-sm-4">
+      <h5>Communications Lead:</h5>
+      <div>
+        <img
+          class="steeering-committee-image"
+          alt="Photo of Alison Lawyer"
+          src="/assets/images/steering-committee/Alison.png"
+        />
+
+        <p class="steeering-committee-name">
+          Alison Lawyer
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <h5>Community Organizers:</h5>
+  <div class="row">
+    <div class="col-sm-4">
+      <img
+        class="steeering-committee-image"
+        alt="Photo of Andrew Tom"
+        src="/assets/images/steering-committee/Andrew.png"
+      />
+
+      <p class="steeering-committee-name">
```

Comment:
It looks like you removed this css class, you could probably remove it from here!

Reply:
You're right 👀 thanks!